pkg/karmadactl/cmdinit/kubernetes/rbac.go

```diff
@@ -68,44 +65,101 @@ func (i *CommandInitOption) CreateClusterRole() error {
 
 	clusterRoleClient := i.KubeClientSet.RbacV1().ClusterRoles()
 
-	clusterRoleList, err := clusterRoleClient.List(context.TODO(), metav1.ListOptions{})
-	if err != nil {
-		return err
-	}
-
-	for _, v := range clusterRoleList.Items {
-		if clusterRole.Name == v.Name {
-			klog.Warningf("ClusterRole %s already exists.", clusterRole.Name)
-			return nil
+	if i.KarmadaClusterProxyAdminRBAC == "true" {
+		// cluster-proxy-admin ClusterRole
+		proxyAdminClusterRole := i.ClusterRoleFromSpec(clusterProxyAdminName, []rbacv1.PolicyRule{
+			{
+				APIGroups: []string{"cluster.karmada.io"},
+				Resources: []string{"clusters/proxy"},
+				Verbs:     []string{"*"},
+			},
+		})
+		err := createIfNotExistClusterRole(clusterRoleClient, proxyAdminClusterRole)
+		if err != nil {
+			return err
```

Comment:
`KarmadaClusterProxyAdminRBAC` should be deployed after `karmada-apiserver` has been deployed. 

https://github.com/karmada-io/karmada/blob/0b9180630d8e3853b83b53e6ca4023fe83f37b33/pkg/karmadactl/cmdinit/kubernetes/deploy.go#L434-L443